.editorconfig

```diff
@@ -5,13 +5,19 @@ charset = utf-8
 indent_size = 4
 indent_style = space
 insert_final_newline = true
-max_line_length = 80
+max_line_length = 120
 tab_width = 8
 trim_trailing_whitespace = true
 
-[*.yml;*.cmake;CMakeLists.txt]
+[*.{cc,h,h.in}]
+max_line_length = 80
+
+[{*.cmake,CMakeLists.txt}]
 indent_size = 2
 tab_width = 2
 
+[*.{html,man,so}]
+max_line_length = 400
```

Comment:
I need a decision what the indent size and max line length shall be for each file.
The editorconfigcheck validates if every file is fitting this configuration.
In the past the line length was strictly 80 for every file.

My suggestion:
- consistent indent=4 for every file
- increase max=120, on reviews check if a line is readable/too complex or not
- for C code we might keep 80, please see commit 5e1f054a for what needs to be changed then
- html,man,so: they need a bigger limit today (i don't wanted to touch troff files myself)

Reply:
I'd been looking at the line length issue myself also. I'm old enough to almost remember actual terminals in use with 80 col limits and normally I don't like to set this low a limit but it really is in keeping with the original coding style. Given how close we are, I agree, let's just keep 80 with C/C++.

For reasons mentioned above, I'm happy with the 120 elsewhere. Consistent indent of 4 (spaces not tabs) works for me too. 

I'm happy to tweak the troff and soelim files. Once we get your changes merged, I could do a follow-up on those. 120 **should** be workable there. I'll take a look. So for now, agreed, let's keep it large.